data/specs.yaml

```diff
@@ -0,0 +1,11 @@
+- version: in-toto Stable (v1.0)
```

Comment:
We should eventually look into having separate fields so that version is exactly the version number not any other qualifiers.